panacea-nft/README.md

```diff
@@ -0,0 +1,76 @@
+# Panacea NFT
```

Comment:
This file is duplicated with the `README.md` in the project root. Please reorganize documents, so that we don't need to keep multiple documents up-to-date.

Reply:
Oh, we will create available functions for each document in the project. Currently, most of the contents are duplicated, so I'll leave it blank for now.

Reply:
I think it's good to have separated README files. But then, the root `README` should just have links to its children.

Reply:
Okay. I'll fix it in another PR.